### DISCUSSION TITLE
Adding statusline capabilities.

### DISCUSSION THREAD

**Discussion by aoi-buh** [nushell/nushell#14381] [Ideas]
Like the Discord discussion [here](https://discord.com/channels/601130461678272522/601130461678272524/987312329047814144) I think having a statusline in nushell would be cool.

**Comment by NotTheDr01ds** [comment]
I agree it would be nice, but I think that features like this are better left to an "outer" program like Tmux or Zellij that can host a shell, provide status-line info (top and/or bottom), and yet still maintain scrollback integrity.  Building something like this into Nushell (or Reedline) would, I believe, require recreating far too many "terminal" features.

Now if we just had a Tmux or Zellij on Windows ...

**Reply by Lockszmith-GH** [reply]
It's [in the works](https://github.com/zellij-org/zellij/pull/2926#issuecomment-2399603414), but it's one person doing the heavy lifting and a community that tests and provides feedback - I recommend following their progress.

**Reply by aoi-buh** [reply]
Yeah maybe it's better for nushell to remain something simpler.
